### ISSUE TITLE
Problem with matcher on Windows platform with latest code

### ISSUE THREAD

**Issue by gafferongames** [networkprotocol/libyojimbo#15]
I'm seeing some strange behavior where mbedtls is receiving a truncated https response from the matcher.go. I haven't seen this before and have no idea why it has started happening today (no changes made to yojimbo). 

I updated to a latest Docker version and noticed this today, so I think it might be related to that.

I'm still investigating and don't know what it is yet so I'm putting this up so people know what's going on if they see this themselves.

Symptoms are:

"premake5 connect" fails to connect and says "failed to connect to matcher, is the matcher running?" even though the matcher *is* running.

Windows only. Seems fine on MacOS

**Comment by gafferongames** [comment]
I'm still getting this. I think Docker has been upgraded a few times since. Still working fine on MacOSX but broken on Windows. I still have no idea what broke this. I'm going to go back to an older version of code (known working on windows), to see if it is still broken there and if not, then binary search to find the point of breakage. WEIRD!

**Comment by gafferongames** [comment]
I'm looking into this issue this week. -- Glenn

**Comment by gafferongames** [comment]
Resolved. The mbedtls example code I had adapted had a bug when the HTTPS response was split into two packets, fixed the code so it could handle multiple calls to mbedtls_ssl_read and the matcher response comes through correctly. I think the split is due to MTU and started happening because of a Docker networking change specific to Windows, however the bug was real, it just never happened on other platforms, because the data came in with one call to ssl_read.
